airgun/entities/host_new.py

```diff
@@ -159,6 +160,23 @@ def get_ansible_roles_modal(self, entity_name):
         self.browser.plugin.ensure_page_safe()
         return view.table.read()
 
+    def enable_tracer(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        view.traces.enable_traces.click()
+        modal = EnableTracerView(self.browser)
+        if modal.is_displayed:
+            modal.confirm.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def get_tracer_title(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.traces.title.read()
```

Comment:
I don't think that this function should be so specific. See my implementation in this work-in-progress PR.
https://github.com/SatelliteQE/airgun/blob/1e2907b322ab3aaa5748167a017fe7277a47f9df/airgun/entities/host_new.py#L202-L206

With this implementation, you can just call `session.host_new.get_traces(host_name)` in your test and just use `title` as you need.

Reply:
This is fine, of course, but in this case there really isn't a reason for it, and making the getter more specific offers better readability in the test.

Reply:
I can't agree with that. Readability is going to be the same in the test.

Reply:
@sambible 
In the test you are going to use something like:
```python
title = session.host_new.get_traces(tracer_install_host.hostname).title
```
Where is better readability compared to:
```python
title = session.host_new.get_tracer_title(tracer_install_host.hostname)
```
?


If my suggestion has the benefit of not adding an ambiguous function to the entities. 
(because `get_traces` function I suggested is going merged and used in the future)

Reply:
I'll go ahead and update then